src/ripple/app/tests/TxQ_test.cpp

```diff
@@ -403,6 +403,33 @@ class TxQ_test : public beast::unit_test::suite
         env(noop(daria), fee(7000), queued);
         env(noop(edgar), fee(7000), queued);
         checkMetrics(env, 5, boost::none, 3, 2, 256);
+        {
+            auto& txQ = env.app().getTxQ();
+            auto aliceStat = txQ.getAccountTxs(alice.id(),
+                env.app().config(), *env.current());
+            if (expect(aliceStat))
+            {
+                expect(aliceStat->size() == 1);
+                expect(aliceStat->begin()->second.feeLevel == 256);
+                expect(aliceStat->begin()->second.lastValid &&
+                    *aliceStat->begin()->second.lastValid == 8);
+                expect(!aliceStat->begin()->second.consequences);
+            }
+
+            auto bobStat = txQ.getAccountTxs(bob.id(),
+                env.app().config(), *env.current());
+            if (expect(bobStat))
+            {
+                expect(bobStat->size() == 1);
+                expect(bobStat->begin()->second.feeLevel = 700 * 256);
```

Comment:
I didn't use the result so it would be more obvious to other developers (and me later) where the numbers came from. I assume that the compiler will do the computation and leave a constant in the executable.